packages/core/src/ahoi_items/QuestionMarkItem.ts

```diff
@@ -0,0 +1,59 @@
+import { ANCHORCLASS, ARROWCLASS, NAVIGATIONCLASS, QUESTIONMARKCLASS } from "../constants";
+import { EOnboardingStages } from "../interfaces";
+import { getColor } from "../utils";
+import { AAhoiItem } from "./AAhoiItem";
+
+export default class QuestionMarkItem extends AAhoiItem {
+    private isOpen: boolean;
+    
+    constructor(parent: HTMLElement, onClick: () => void) {
+      super(parent, QUESTIONMARKCLASS, onClick, {iconClass: "fa-question", htmlTitle: "Open Onboarding", stage: null});
+      this.isOpen = false;
+    }
+  
+    protected clickHandler() {
+      let delChild: Element[] = [];
+      const navItems = document.querySelectorAll(`.${NAVIGATIONCLASS}`);
```

Comment:
```suggestion
      const navItems = parent.querySelectorAll(`.${NAVIGATIONCLASS}`);
```

`document.querySelectorAll()` will cause side effects if VisAhoi is used multiple times on the same page.

Maybe `parent` works, but it would be better that a separate Navigation class manages the state and show/hide of children.